static/bidder-params/connatix.json

```diff
@@ -8,6 +8,10 @@
         "type": "string",
         "minLength": 1,
         "description": "Placement ID"
+      },
+      "viewabilityPercentage": {
+        "type": "number",
+        "description": "Declared viewability percentage (values from 0 to 1, where 1 = 100%)"
```

Comment:
If `viewabilityPercentage`'s value should range from 0 to 1, should we specify a `minimum` and a `maximum`?
```
12       "viewabilityPercentage": {
13         "type": "number",
14         "description": "Declared viewability percentage (values from 0 to 1, where 1 = 100%)"
   +       "minimum": 0,
   +       "maximum": 1
15       }
16     },
static/bidder-params/connatix.json
```
If not, is a validation inside `adapters/connatix/connatix.go` desired?

Reply:
I don’t want to enforce this because is handled on our side.